src/cr/cube/matrix/measure.py

```diff
@@ -524,6 +545,15 @@ def _subtotal_rows(self):
         return SumSubtotals.subtotal_rows(self._base_values, self._dimensions)
 
 
+class _Sums(_BaseSecondOrderMeasure):
+    """Provides the sum measure for a matrix."""
+
+    @lazyproperty
+    def blocks(self):
+        """2D array of the four 2D "blocks" making up this measure."""
+        return NanSubtotals.blocks(self._cube_measures.cube_sum.sums, self._dimensions)
+
```

Comment:
Not sure about the `NanSubtotals` bit here, better check with Mike.

Reply:
yes sums should add